src/main/java/us/abstracta/wiresham/Flow.java

```diff
@@ -45,6 +48,7 @@ public class Flow {
       .put("!server", SendPacketStep.class)
       .put("!client", ReceivePacketStep.class)
       .put("!include", IncludePacketStep.class)
+      .put("!parallel:", ParallelPacketStep.class)
```

Comment:
is it expected to include colons here?

Reply:
I thought we agree on the design of the yaml to be something like:
```yaml
- !server { data: 48656C6C6F, port: 23}
- !client { data: 48656C6C6F2C2049276D204A6F686E }
- !parallel:
    -
      - !server { data: 48656C6C6F204A6F686E, port: 24 }
      - !client { data: 48656C6C6F2C2049276D204A6F686E }
      - !server { data: 427965204A6F686E }
    - 
      - !server { data: 48656C6C6F, port: 25}
      - !client { data: 48656C6C6F2C2049276D204A6F686E }
     ```
 However, it does not complain the standards we had before. 
Brackets is the standard we used. Therefore yaml will look something like:
```yaml
- !server { data: 48656C6C6F, port: 23}
- !client { data: 48656C6C6F2C2049276D204A6F686E }
- !parallel {
    -
      - !server { data: 48656C6C6F204A6F686E, port: 24 }
      - !client { data: 48656C6C6F2C2049276D204A6F686E }
      - !server { data: 427965204A6F686E }
    - 
      - !server { data: 48656C6C6F, port: 25}
      - !client { data: 48656C6C6F2C2049276D204A6F686E }
   }
   ```
That looks aligned with previous yaml syntax. What do you think?

Reply:
I wonder now if the first alternative "aligns" with yaml. Would need to review. I initially thought that both alternatives where mainly the same, but it may not be the case. Let's research a little about that.